src/fusion.cpp

```diff
@@ -158,6 +160,98 @@ static auto AllocateBuffersAndMakeConvBiasActivFusionInvokeParams(
                                               gfx90aaltimpl);
 }
 
+namespace {
+
+int GetFusionMode(const miopenFusionPlanDescriptor_t fusePlanDesc)
+{
+    int fusion_mode = -1;
```

Comment:
This can be moved to `driver_arguments.cpp`

Reply:
moved to `driver_arguments.cpp`